### PR TITLE
Add Popup Html Structure

### PR DESCRIPTION
**Pull request by saadat-ali-awan**
### Project requirements

- You should implement the following interactions:
  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.
- You should implement popups for both mobile and desktop screen sizes.

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -499,6 +594,53 @@ footer {
...
+    padding: 24px;
+    height: calc(100vh - 244px);
+    width: calc(100vw - 520px);
+    top: 122px;
+    left: 260px;
+    display: grid;
+    grid-template-columns: 60% auto 40px;
+    grid-template-rows: auto auto 28px auto auto;
+  }
+
```
Comment (mateo951) [inline main.css:643]: There are a few issues with your popup on desktop sizes. Kindly make sure that you follow as much as possible Figma's design 🧐 We could work on the font, text color, and margins of the popup to resemble the design! You are on the right path 😎 One last thing, the color of the x button is too light and loses itself sometimes through the design. Please make the necessary changes in order to meet the project requirements 😉

### Your design 
image

### Figma'
Captura
s

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi Team, 

👌 **Your project highlights** 😎 
- Linters are set correctly ✔
- GitHub Flow implemented 👌
- JS functionality

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by sheylaPozo** [comment]
## Status: ✔️ Project Approved ✔️ 
----

Hello @Saadat123456 🔥 

> # Excellent work on your project, it looks very professional and it's completed. Congrats on the good work!

surprised

Now you can celebrate! 🥇 🚀 👍 💯 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️
`There's nothing left to say but time to merge! 😊`

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sheylaPozo** [comment]
## Optional Suggestions 
---

- [x] Would look better if you re-check that the `see project` button works in your last update of deployment, I've tried to run the project and open the live link and press the `see project` button but hasn't shown me the pop-up window, I know it might be that the last update deployed branch is behind the deployed project, so just in case try to deploy it again with your last made changes, and run the project checks to see if all works properly, since this project you will use it until the end of Microverse and all have to run correctly. Since this is an optional suggestion, doesn't hold you back from approval, good job doing all the required changes, your project's been approved. ✔😊
check👍 💯 

tenor

**Comment by sheylaPozo** [comment]
@Saadat123456  When I reviewed it, I saw all the checks were running, kindly fix the linter errors, senior reviewers will check, and it seems like your pop up window is still not functioning and there's a linter error, you can fix with the auto-fix option `--fix`

**Comment by sheylaPozo** [comment]
### Optional changes ♻

@Saadat123456 When I reviewed it, I saw all the checks were running, kindly fix the linter errors, senior reviewers will check for conformities and notice there's a linker error, you can fix it with the auto-fix option --fix or go one by one manually until all the linter errors are corrected 😁📌

**Comment by sheylaPozo** [comment]
### Optional changes ♻

@Saadat123456 When I reviewed it, I saw all the checks were running so I proceed to approve, seems like there was some change that caused a linter error before you merged into main, kindly fix the linter errors, senior reviewers will check for conformities and notice there's a linter error, you can fix it with the auto-fix option `--fix` or go one by one manually until all the linter errors are corrected 😁📌

**Pull request merged**